app/extras/poll_exporter.rb

```diff
@@ -0,0 +1,121 @@
+class PollExporter
+
+  include Routing
+
+  def initialize (poll)
+    @poll = poll
+  end
+
+  def file_name
+    "#{@poll.title.parameterize}-export.csv"
+  end
+
+  def poll_info (current_user)
+     PollEmailInfo.new(recipient: current_user, event: @poll.created_event, action_name: :export)
+  end
+
+  def meta_table
+    outcome = @poll.current_outcome
+    engagement = nil
+    I18n.with_locale(:en) {
```

Comment:
No need for this with_locale; all of the requests are wrapped in a with_locale call for the current user, so this is just forcing :en on folks who may not want it.

```
I18n.t(:"export.poll.percent_voted",
  num: @poll.stances_count,
  denom: @poll.members.count,
  percent: "#{100 * @poll.stances_count / @poll.members.count}%"
)
```

I'd also be tempted to write an `engagement` method on poll:
```
# poll.rb
def engagement
  @engagement ||= "#{100 * stances_count / members.count}%"
end
```